bin/riak-bucket-exporter.js

```diff
@@ -22,9 +23,16 @@ var count = 0;
 var openWrites = 0;
 var db = require("riak-js").getClient({host: program.host, port: program.port});
 var fs = require('fs');
+var deleteKeys = false;
```

Comment:
okay. if this is not too confusing for you we could do a 

`var deleteKeys = !program.import && !!program.delete;`

but I mean we would still need the block below to write the output. So we can also leave it this way.